src/gwbot/modules/003_admin/admin.c

```diff
@@ -36,18 +36,20 @@ int GWMOD_SHUTDOWN_DEFINE(003_admin, struct gwbot_state *state)
 
 typedef enum _mod_cmd_t {
 	CMD_NOP		= 0u,
-	ADM_CMD_BAN	= (1u << 0u),
-	ADM_CMD_UNBAN	= (1u << 1u),
-	ADM_CMD_KICK	= (1u << 2u),
-	ADM_CMD_WARN	= (1u << 3u),
-	ADM_CMD_MUTE	= (1u << 4u),
-	ADM_CMD_TMUTE	= (1u << 5u),
-	ADM_CMD_UNMUTE	= (1u << 6u),
-	ADM_CMD_PIN	= (1u << 7u),
-	ADM_CMD_UNPIN	= (1u << 8u),
-	ADM_CMD_DELETE	= (1u << 9u),
-	USR_CMD_REPORT	= (1u << 10u),
-	USR_CMD_DELVOTE	= (1u << 11u),
+	ADM_CMD_BAN	= (1u << 1u),
```

Comment:
Shouldn't we start from `(1u << 0u)`?

Reply:
Oh right, my bad, will push a revision for this.